### DISCUSSION TITLE
Duplicating an item and its related items

### DISCUSSION THREAD

**Discussion by adanielyan** [directus/directus#8988] [Archive]
Currently when I duplicate an item with a one to many relationship, that relationship is removed from original item and is assigned to the new item. How can I configure the Duplication so that the related items are also duplicated? E.g. when I duplicate (save as) a Post all related Comments should be duplicated too and a relationship between the new Post item and duplicated Comments is created.

Is it possible to setup such a behavior?

**Comment by benhaynes** [comment]
Check the duplication section of the collection under settings. That should allow you to control what gets duplicated.

**Comment by adanielyan** [comment]
Hi Ben, thank you for a quick response. I did check the Duplication section. But couldn't find a combination of selections that would also duplicated the related items. I don't want to duplicate the relationship, but rather the related item itself and create a new relationship between the duplicated items. 

In duplication settings I tried selecting the related field on One side, on Many side, as well as on both sides. But none of these combinations duplicated the related item.
